### PR TITLE
Hw5: Task 2

### PR DESCRIPTION
**Pull request by ValeriL**

### CODE REVIEW

File: Homework/homework5/task2.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Ожидаемый результат:
+print(custom_sum.__doc__)  # 'This function can sum any objects which have __add___'
+print(custom_sum.__name__)  # 'custom_sum'
+print(custom_sum.__original_func)  # <function custom_sum at <some_id>>
+"""
+
+from typing import Callable
+
+
+def wraps(wrapped: Callable) -> Callable:
```
Comment (Cjkjvfnby) [inline Homework/homework5/task2.py:21]: can you find an analog in the standard library?

File: Homework/homework5/test_task2.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+def test_addition_of_original_func_attribute():
+    @print_result
+    def f(a=5):
+        """Any doc information."""
+        return a
+
+    not_original_func = f.__original_func
+
+    assert not_original_func() == f()
```
Comment (Cjkjvfnby) [inline Homework/homework5/test_task2.py:22]: You test that result of one function is equals to result of the another function.   This does not matter that they are equals.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
Does it print?

File: Homework/homework5/task2.py (hunk 1 above)
Comment (ValeriL) [reply Homework/homework5/task2.py:21]: Yes, https://docs.python.org/3/library/functools.html#functools.wraps

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**